modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/pom.xml

```diff
@@ -25,27 +26,38 @@
 			<version>2.0.0</version>
 			<scope>provided</scope>
 		</dependency>
-#if (${liferayVersion.startsWith("7.0")})
 		<dependency>
 			<groupId>com.liferay.portal</groupId>
 			<artifactId>com.liferay.portal.kernel</artifactId>
 			<version>2.0.0</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
```

Comment:
Do we need both `osgi.cmpn` and `org.osgi.service.component.annotations` ? Can we just add one or the other?

Reply:
Hey @gamerson,

Apparently, won't be a problem use only the `org.osgi.service.component.annotations`. I'll do this change and test to make sure everything is fine. 😉